src/nl/rubensten/texifyidea/inspections/latex/LatexFileNotFoundInspection.kt

```diff
@@ -46,33 +46,36 @@ open class LatexFileNotFoundInspection : TexifyInspectionBase() {
                 val fileArgument = arguments[i] as? RequiredFileArgument ?: continue
                 val extensions = fileArgument.supportedExtensions
                 val parameter = parameters[i]
-                val fileName = parameter.requiredParam?.firstChildOfType(LatexNormalText::class)?.text ?: continue
+                val fileNames = parameter.requiredParam?.firstChildOfType(LatexNormalText::class)?.text?.split(',') ?: continue
```

Comment:
An idea to make a util to split contents of a `LatexParameter`? I see this getting used in the future more often (also with optional parameters) and makes it a bit cleaner.

BTW, some time we should refactor this kind of code constructs to `command.requiredParameter(0)`